examples/notebooks/lambdify.ipynb

```diff
@@ -0,0 +1,810 @@
+     "collapsed": false,
+     "input": [
+      "fn_taylor = fn.series(xs,0,6).removeO()\n",
+      "fn_taylor"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "latex": [
+        "$$\\frac{x^{5}}{120} + \\frac{x^{4}}{24} + \\frac{x^{3}}{6} + \\frac{x^{2}}{2} + x + 1$$"
+       ],
+       "metadata": {},
+       "output_type": "pyout",
+       "prompt_number": 21,
+       "text": [
+        "  5    4    3    2        \n",
+        " x    x    x    x         \n",
+        "\u2500\u2500\u2500 + \u2500\u2500 + \u2500\u2500 + \u2500\u2500 + x + 1\n",
+        "120   24   6    2         "
+       ]
+      }
+     ],
+     "prompt_number": 21
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Using the `lambdify()` function and timing the operation for both cases"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "fn_taylor_lambdified = lambdify(xs, fn_taylor, modules=['numpy'])\n",
+      "%timeit fn_taylor_lambdified(x)"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "output_type": "stream",
+       "stream": "stdout",
+       "text": [
+        "1000 loops, best of 3: 265 \u00b5s per loop\n"
+       ]
+      }
+     ],
+     "prompt_number": 22
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "**Result: 265 \u00b5s**"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%timeit ans = [fn_taylor.subs(xs, i) for i in x]"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [
+      {
+       "output_type": "stream",
+       "stream": "stdout",
+       "text": [
+        "1 loops, best of 3: 1.61 s per loop\n"
+       ]
+      }
+     ],
+     "prompt_number": 23
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "**Result: 1.61 s**\n",
+      "\n",
+      "##Creating a plot to contrast the difference in runtime"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "%matplotlib inline\n",
+      "import matplotlib.pyplot as plt"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 24
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Creating a list of the runtimes from the `%timeit` commands"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "collapsed": false,
+     "input": [
+      "xl = [1,2,3,4,5]\n",
+      "yl = [3.96, 13.8, 98.6, 179, 256]\n",
+      "ynl = [188000, 534000, 942000, 1270000, 1610000]"
+     ],
+     "language": "python",
+     "metadata": {},
+     "outputs": [],
+     "prompt_number": 25
+    },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "Generating the plots with different scales for each case. "
```

Comment:
I guess it will be better that you plot the scaling factor rather than the wall time of running the program. The absolute wall time is useless as it specific to your machine at that time. It can change drastically with other machines and other setups.